### PR TITLE
chore: A full, minimal, reproducible example of the RBAC feature

### PR DESCRIPTION
**Pull request by dmartinol**
# What this PR does / why we need it:
A guided series of notebooks to showcase the RBAC feature in a local environment.
* Build an OIDC server
* A Feast sample repo
* A distributed Feast servers topology
* A test client to validate the RBAC permissions for the configured users

# Which issue(s) this PR fixes:
Second part of #4450

### CODE REVIEW

**Comment by dmartinol** [comment]
@franciscojavierarceo @tokoko

**Comment by dmartinol** [comment]
@tokoko IT failed with quota errors, should we give it another run?
```
ERROR sdk/python/tests/integration/online_store/test_python_feature_server.py::test_push[ParameterSet(values=(LOCAL:File:bigtable:python_fs:False,), marks=[], id=None)] - google.api_core.exceptions.ResourceExhausted: 429 Quota exceeded for quota metric 'Table metadata write requests' and limit 'Table metadata write requests per minute per user' of service 'bigtableadmin.googleapis.com' for consumer 'project_number:883187091528'. [reason: "RATE_LIMIT_EXCEEDED"
```

File: examples/rbac-local/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# RBAC demo
+RBAC demo with local environment.
+
+## System Requirements
+* Clone of the Feast repo
+* Docker
+* yq
```
Comment (lokeshrangineni) [inline examples/rbac-local/README.md:7]: what is `yq`

Comment (dmartinol) [reply examples/rbac-local/README.md:7]: I thought it was not worth to mention it 😮 
[a lightweight and portable command-line YAML, JSON and XML processor ](https://github.com/mikefarah/yq)

Comment (lokeshrangineni) [reply examples/rbac-local/README.md:7]: may be better to mention it as a link. Hope i am not an ignorant if it is a popular library.
* [yq](https://github.com/mikefarah/yq)

**Review by franciscojavierarceo — APPROVED** [review]
nice!

**Pull request merged**